russian/invite-proffesor/invite-proffesor.md

```diff
@@ -0,0 +1,8 @@
+Профессор Воробьев ответил на ваш звонок почти сразу.
+К сожалению, он не успел закончить настройку телескопа после установки на него новой мощной линзы.
+Поэтому оговоренный эксперимент придется отложить на следующий день.
+
+Тут ваш взгляд падает на полки, уставленные банками с медом и банками со сливками.
+Нежные воздушные взбитые сливки... И тут вас осеняет! Вы включаете свой смартфон. Открываете "Контакты", группу "Длинные ножки". И приглашаете к себе трех подруг.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.